.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
```

Comment:
Without `/` maybe?

Reply:
This is what CRA does by default. What is the difference, I have no idea?

Reply:
Ah so nvm then!